dbt/adapters/databricks/python_submissions.py

```diff
@@ -287,7 +300,7 @@ def get_elapsed() -> float:
 
         while get_elapsed() < MAX_CLUSTER_START_TIME:
             status_response = self.get_cluster_status()
-            if status_response.get("state") == "RUNNING":
+            if status_response.get("state").lower() == "running":
```

Comment:
hehe clever 😉

Reply:
How much should we trust the api generally?  Like, do I need to provide a default in case state is not present?

Reply:
I wouldn't use a default because if the api changes without notice it's probably best to fail early.